test/integration/filters/set_response_code_filter_config.proto

```diff
@@ -8,4 +8,5 @@ message SetResponseCodeFilterConfig {
   string prefix = 1;
   uint32 code = 2 [(validate.rules).uint32 = {lt: 600 gte: 200}];
   string body = 3;
+  bool is_terminal_filter = 4;
```

Comment:
I would do this in a different test filter instead of overloading the existing one

Reply:
reset.